spec/factories/rents.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :rent do
+    user
+    book
+    start_date { '2018-09-24' }
+    end_date { '2018-09-24' }
```

Comment:
Try to set the end_date relative to the random start_date selected by Faker. This could be, for example 5 days after the start_date.